_includes/tutorials/session-windows/ksql/code/tutorial-steps/dev/check-rowtime.sql

```diff
@@ -1 +1,2 @@
+SET CLI COLUMN-WIDTH 20;
```

Comment:
btw, in 390ae243fa88ea0cbaf4ed3eea09f94975420fd9, we added support for defining column width for the set of commands in `docker_ksql_cli_session` blocks.
e.g. https://github.com/confluentinc/kafka-tutorials/blob/cdd3747b244aa6537eb6d20b054591edfc7789d7/_data/harnesses/aggregating-count/ksql.yml#L38. This executes the `SET CLI` command behind the scenes.

This approach works, too, however.